src/ForceRemoveModal.jsx

```diff
@@ -6,7 +6,7 @@ import cockpit from 'cockpit';
 
 const _ = cockpit.gettext;
 
-const ForceRemoveModal = ({ name, reason, handleForceRemove }) => {
+const ForceRemoveModal = ({ name = "intermediate image", reason, handleForceRemove }) => {
```

Comment:
I'm not sure whether I should translate it, it's used in `cockpit.format(_("Delete $0?"), name)` down below. It is automatically translated when used this way, right? I'm confused.